packages/qt5-qtbase/postinst

```diff
@@ -23,7 +23,7 @@ fi
 ## FIXME: Qt should be built with fontconfig somehow instead
 ## of using direct path to fonts.
 ## Currently, using post-installation script to create symlink
-## from /system/bin/fonts to $PREFIX/lib/fonts if possible.
+## from /system/bin/fonts to @TERMUX_PREFIX@/lib/fonts if possible.
```

Comment:
Unnecessary.

Reply:
hm, It's chsnged automatically, Will modify it manually